controller/konnect/ops/ops_kongpluginbinding.go

```diff
@@ -262,8 +262,11 @@ func kongPluginBindingToSDKPluginInput(
 	}
 
 	pluginConfig := map[string]any{}
-	if err := json.Unmarshal(plugin.Config.Raw, &pluginConfig); err != nil {
-		return nil, err
+	if rawConfig := plugin.Config.Raw; rawConfig != nil {
```

Comment:
I think it worth a detailed comment to explain that empty `plugin.Config.Raw` is valid but it would make the unmarshal fail so we need to skip the unmarshal when it is empty.

Reply:
Changed the comment to mention it's a valid case.